airflow/dags/deploy_dbt_docs/sync_dbt_docs_to_metabase.yml

```diff
@@ -25,19 +26,21 @@ env_vars:
   BIGQUERY_KEYFILE_LOCATION: /secrets/jobs-data/service_account.json
   DBT_PROJECT_DIR: /app
   DBT_PROFILE_DIR: /app
+  DBT_DATABASE: "{{ get_project_id() }}"
   DBT_TARGET: "{{ env_var('DBT_TARGET') }}"
+  MB_HOST: dashboards.calitp.org
 
 secrets:
   - deploy_type: volume
     deploy_target: /secrets/jobs-data/
     secret: jobs-data
     key: service-account.json
   - deploy_type: env
-    deploy_target: METABASE_USER
+    deploy_target: MB_USER
```

Comment:
Just curious about the reason for changing this?

Reply:
Ah dbt-metabase will pick up the new ones directly, rather than us getting them from `os.environ` and passing as CLI args in the subprocess call.